development-kit/pkg/enums/engine/advisories/leaks/regular/regular.go

```diff
@@ -461,3 +461,19 @@ func NewLeaksRegularPasswordExposedInHardcodedURL() text.TextRule {
 		},
 	}
 }
+
+func NewLeaksRegularWPConfig() text.TextRule {
+	return text.TextRule{
+		Metadata: engine.Metadata{
+			ID:          "abc",
```

Comment:
Please add a uuid v4 in ID Field how to `5332204a-0d3d-4fe3-a73b-29525101afa0`

Reply:
This ID field represents a unique identifying key for the vulnerability.

Reply:
Okay, but how do you generate this ID? probably I'll add more rules and know the process made things easy, then I dont net do ask for ID every time.

Reply:
Yeah, You can generate an random uuid v4 of your choice, we get always in https://www.uuidgenerator.net/